packages/core/chrome/core-chrome-browser-internal/src/ui/project/header.tsx

```diff
@@ -171,11 +172,16 @@ export const ProjectHeader = ({
   docLinks,
   ...observables
 }: Props) => {
+  const isVisible = useObservable(observables.isVisible$, false);
   const [navId] = useState(htmlIdGenerator()());
   const [isOpen, setIsOpen] = useLocalStorage(LOCAL_STORAGE_IS_OPEN_KEY, true);
   const toggleCollapsibleNavRef = createRef<HTMLButtonElement & { euiAnimate: () => void }>();
   const headerActionMenuMounter = useHeaderActionMenuMounter(observables.actionMenu$);
 
+  if (!isVisible) {
+    return <HeaderTopBanner headerBanner$={observables.headerBanner$} />;
```

Comment:
This follows the code used for the classic header behavior:
https://github.com/elastic/kibana/blob/cf285a71daaed84b0ea656316d14d83bf9df927b/packages/core/chrome/core-chrome-browser-internal/src/ui/header/header.tsx#L102-L109

One difference is that I'm not adding the loading indicator. If we need it, I can add the header logo component which functions as a loading indicator in the project layout. 

I'm not sure why it doesn't just return empty content. It could be for accessibility.